Firstturnmedia/cmbd8
# CH-529: Add entity ref or text override module.

To test:

* Enable the module
* Add a field to an entity of type "Entity Reference or Text".
* Edit the entity. The widget should have side-by-side fields for the entity autocomplete, and the override text.
